batch/batch/worker/worker.py

```diff
@@ -1012,15 +1110,8 @@ async def upload_log(self):
             await self.get_log(),
```

Comment:
Ah, hmm, uploading the log requires loading the entire log into RAM. As a follow up PR, let's fix this so that we either use the aiotools Copier or we just manually read and write a fixed size block at a time.